modules/nf-core/amulety/esm2/main.nf

```diff
@@ -0,0 +1,51 @@
+process AMULETY_ESM2 {
+    tag "$meta.id"
+    label 'process_medium'
+    label 'process_gpu'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/amulety_igblast:b2a7736f645c40e5':
+        'community.wave.seqera.io/library/amulety_igblast:659eaa872785adeb' }"
+
+    input:
+    tuple val(meta), path(tsv)
+    val(chain)
+
+    output:
+    tuple val(meta), path("*_esm2.tsv"), emit: embedding
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    export TRANSFORMERS_CACHE="./cache"
+    amulety \\
```

Comment:
```suggestion
    TRANSFORMERS_CACHE="./cache" amulety \\
```
This might cause some issues later, better to only apply this variable to the command that needs it